cmd/controller/main.go

```diff
@@ -192,12 +199,17 @@ func main() {
 	flag.BoolVar(&profiling, "enable-profiling", false, "when set, enables the endpoints on the http server under /debug/pprof/")
 	flag.BoolVar(&externalCloudProvider, "external-cloud-provider", false, "when set, kubelets will receive --cloud-provider=external flag")
 	flag.DurationVar(&skipEvictionAfter, "skip-eviction-after", 2*time.Hour, "Skips the eviction if a machine is not gone after the specified duration.")
+	flag.StringVar(&nodeHTTPProxy, "node-http-proxy", "", "If set, this proxy will be configured on all nodes.")
+	flag.StringVar(&nodeNoProxy, "node-no-proxy", ".svc,.cluster.local,localhost,127.0.0.1,127.0.0.0/", "If set, this proxy will be configured on all nodes.")
```

Comment:
@thz PTAL

Reply:
there is a trailing slash probably meant to be `/8`.

Reply:
proxy authentication is handled as url-embedded credentials, right?

Reply:
Yes